README.md

```diff
@@ -1 +1,91 @@
-# generator-core-ibm-golang-gin
+# IBM Core Golang Gin Generator
+
+[![IBM Cloud powered][img-ibmcloud-powered]][url-cloud]
+[![Travis][img-travis-master]][url-travis-master]
+
+[img-ibmcloud-powered]: https://img.shields.io/badge/IBM%20Cloud-powered-blue.svg
+[url-cloud]: http://bluemix.net
+
+[img-travis-master]: https://travis.ibm.com/arf/generator-core-golang-gin.svg?token=sXo91qAvkL12pygC7Sta&branch=master
```

Comment:
these arf urls need to be updated

Reply:
Could you add travis to the configuration of this repo so I can get the correct links?